<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>633</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>TexitEasy</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuOuvrir_R_cent">
     <property name="title">
      <string>Ouvrir Récent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionDeleteLastOpenFiles"/>
    </widget>
    <widget class="QMenu" name="menuEncodage">
     <property name="title">
      <string>Enregistrer avec l'encodage</string>
     </property>
     <addaction name="actionSaveWithUTF8"/>
     <addaction name="separator"/>
     <addaction name="actionSaveWithOtherEncoding"/>
    </widget>
    <widget class="QMenu" name="menuReopenWithEncoding">
     <property name="title">
      <string>Réouvrir avec l'encodage</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuTexDirectives">
     <property name="title">
      <string>Directives Tex</string>
     </property>
     <addaction name="actionTexDirProgram"/>
     <addaction name="actionTexDirEncoding"/>
     <addaction name="actionTexDirSpellChecker"/>
     <addaction name="actionTexDirRoot"/>
    </widget>
    <addaction name="actionNouveau"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOuvrir_R_cent"/>
    <addaction name="menuReopenWithEncoding"/>
    <addaction name="actionOpenLastSession"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="menuEncodage"/>
    <addaction name="separator"/>
    <addaction name="menuTexDirectives"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Compiler</string>
    </property>
    <widget class="QMenu" name="menuOtherBuilder">
     <property name="title">
      <string>Autre compilation</string>
     </property>
    </widget>
    <addaction name="actionDefaultCommandLatex"/>
    <addaction name="actionBibtex"/>
    <addaction name="actionClean"/>
    <addaction name="separator"/>
    <addaction name="menuOtherBuilder"/>
    <addaction name="separator"/>
    <addaction name="actionAutoView"/>
    <addaction name="actionView"/>
    <addaction name="actionLinkSync"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuTh_me">
     <property name="title">
      <string>Thème</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionOpenConfigFolder"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menuTh_me"/>
    <addaction name="actionSendFeedback"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFindReplace"/>
    <addaction name="actionAutoIndent"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPdfViewerInItsOwnWidget"/>
    <addaction name="actionSplitEditor"/>
   </widget>
   <widget class="QMenu" name="menuPdf">
    <property name="title">
     <string>Pdf</string>
    </property>
    <addaction name="actionOpenPdf"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuBuild"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuPdf"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Enregistrer Sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDefaultCommandLatex">
   <property name="text">
    <string notr="true">defaultCommandLatex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Préférences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAutoView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Afficher automatiquement le pdf</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Refaire</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Couper</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
  <action name="actionFindReplace">
   <property name="text">
    <string>Rechercher / Remplacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>Voir le pdf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOpenConfigFolder">
   <property name="text">
    <string>Ouvrir le dossier thème</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDeleteLastOpenFiles">
   <property name="text">
    <string>Supprimer l'historique</string>
   </property>
  </action>
  <action name="actionBibtex">
   <property name="text">
    <string>Bibtex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A propos de TexitEasy</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionLinkSync">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Synchroniser le pdf</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionOpenLastSession">
   <property name="text">
    <string>Ouvrir les derniers documents</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Nettoyer</string>
   </property>
  </action>
  <action name="actionPdfViewerInItsOwnWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Afficheur pdf séparé</string>
   </property>
  </action>
  <action name="actionSplitEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Split the editor</string>
   </property>
  </action>
  <action name="actionOpenPdf">
   <property name="text">
    <string>Open Pdf</string>
   </property>
  </action>
  <action name="actionSaveWithUTF8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionOpenWithUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionOpenWithISO_8859_1">
   <property name="text">
    <string>ISO 8859-1</string>
   </property>
  </action>
  <action name="actionSaveWithOtherEncoding">
   <property name="text">
    <string>Autre</string>
   </property>
  </action>
  <action name="actionTex_directives">
   <property name="text">
    <string>Tex directives</string>
   </property>
  </action>
  <action name="actionTexDirProgram">
   <property name="text">
    <string>Compilateur</string>
   </property>
  </action>
  <action name="actionTexDirEncoding">
   <property name="text">
    <string>Encodage</string>
   </property>
  </action>
  <action name="actionTexDirSpellChecker">
   <property name="text">
    <string>Langage</string>
   </property>
  </action>
  <action name="actionTexDirRoot">
   <property name="text">
    <string>Fichier Racine</string>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="text">
    <string>Envoyer votre avis</string>
   </property>
  </action>
  <action name="actionAutoIndent">
   <property name="text">
    <string>Indenter automatiquement</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
